drivers/ethernet/eth_stm32_hal.c

```diff
@@ -26,10 +26,22 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 #include "eth_stm32_hal_priv.h"
 
+
+#if defined(DT_DTCM_BASE_ADDRESS)
```

Comment:
Even if this is the only known way we currently have to get ethernet back on STM32F7, I'd prefer to see this as a Kconfig option. Eventually, we'll get the the __nocache option working as well and users will be able to choose the option they prefer.

Reply:
Done